src/collar/oc_rlvsuite.lsl

```diff
@@ -81,7 +81,7 @@ list g_lUtilityNone = ["BACK"];
 
 
 list g_lRLVList = [   // ButtonText, CategoryIndex, RLVCMD, Auth
-    "Emote"         , 0 , "emote"                               , 15 ,  // 1
+    "EmoteTrunc"    , 0 , "emote"                               , 15 ,   // 1
```

Comment:
Looks fine. The @emote command truncates, and there isnt a simple way (without redirect) to actually prevent emote. Which arguably belongs in a gag anyway/